### PR TITLE
Create swedish.properties

### PR DESCRIPTION
**Pull request by ItsLynix**
Added swedish translation

### CODE REVIEW

File: src/languages/swedish.properties [hunk 1]
```diff
@@ -0,0 +1,272 @@
+# Properties for the swedish translation
+# Contributors: ItsLynix
+ 
+REMOVED_WORLD_SELECTION=Du har tagit bort ditt val av värld. Arbeta med den aktuella världen (om möjligt).
+CREATED_GROUP=Du har skapat en grupp som heter %s
+DELETED_GROUP=Du har tagit bort en grupp som heter %s, dess användare har standardgruppen nu.
+ 
+ERROR_GROUP_ALREADY_EXISTS='%s' group already exists\!
+ERROR_GROUP_DOES_NOT_EXIST='%s' group does not exist\!
```
Comment (l1ttleO) [inline src/languages/swedish.properties:9]: Needs to be translated

File: src/languages/swedish.properties [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+ERROR_DESTINATION_HIGHER_THAN_YOURS=Destinationsgruppen kan inte vara densamma som din eller högre.
+ERROR_SUBGROUP_HIGHER_THAN_YOURS=Undergruppen kan inte vara densamma som din eller högre.
+ERROR_NEW_GROUP_NOT_HIGHER=Den nya gruppen måste ha en högre rang än den nuvarande.
+ERROR_NEW_GROUP_NOT_LOWER=Den nya gruppen måste ha en lägre rang än den nuvarande.
+ERROR_PLAYERS_NOT_MEMBERS_OF_GG=Spelare kan inte vara medlemmar i GlobalGroups direkt.
+ERROR_YOU_DO_NOT_INHERIT=Du kan inte ändra en spelare som involverar en grupp som du inte ärver.
+ERROR_CANT_REMOVE_PERMISSION=Du kan inte ta bort en behörighet som du inte har: '%s'
+ERROR_CANT_ADD_PERMISSION=Du kan inte lägga till en behörighet som du inte har: '%s'
+ 
+ERROR_BUKKIT_INVALID_ARGUMENTS=Bukkit gav ogiltiga argument array\! Cmd: %s args.length: %d
```
Comment (l1ttleO) [inline src/languages/swedish.properties:60]: Translate "argument array"

File: src/languages/swedish.properties [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+COMMAND_NOT_PERMITTED=Du får inte använda det kommandot.
+DEFAULT_WORLD_SELECTED=Inte kunnat hämta din värld. Standardvärlden %s valdes.
+WORLD_SELECTION_NEEDEDED=Kan inte hämta din värld. Det behövs ett val av värld.
+USE_MANSELECT=Använd /manselect <värld>
+PLAYER_NOT_FOUND=Spelaren hittades inte\!
+TOO_MANY_MATCHES=För många matcher hittades\!
+
+# Meddela textelement.
+PLAYER_WAS=\ var
+OU_WERE=Du var
```
Comment (l1ttleO) [inline src/languages/swedish.properties:141]:
```suggestion
YOU_WERE=Du var
```

File: src/languages/swedish.properties [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+WorldDatHolder.WARN_USING_DEFAULT=Användning av standardvärden:
+WorldDatHolder.ERROR_NO_DEFAULT=Det fanns ingen standardgrupp deklarerad i filen: %s
+WorldDatHolder.WARN_INHERITED_NOT_FOUND=Inherited group '%s' not found för group %s. Ignorerar posten i filen: %s
+WorldDatHolder.ERROR_INVALID_NODE_USER=Invalid node for user entry #(%d) in file: %s
+WorldDatHolder.ERROR_INVALID_FORMAT_FOR_USER=Invalid formating found for user: %s in file: %s
+WorldDatHolder.ERROR_DUPLICATE_USER=Denna användare har deklarerats mer än en gång: %s i filen: %s
+WorldDatHolder.ERROR_INVALID_FORMAT_IN_USER=Invalid formating found in '%s' for user: %s in file: %s
+WorldDatHolder.ERROR_UNKNOWN_ENTRY_USER=Unknown entry found in '%s' for user: %s in file: %s
+WorldDatHolder.WARN_NO_GROUP_STATED=Det finns ingen grupp %s, som anges för spelare %s: Ställ in "%s" för filen: %s
+WorldDatHolder.WARN_INVALID_SUBGROUP=Invalid Subgroup data for user: %s. Ignorerar posten i filen: %s
```
Comment (l1ttleO) [inline src/languages/swedish.properties:169]: There's a couple of untranslated strings

File: src/languages/swedish.properties [hunk 5]
```diff
@@ -0,0 +1,272 @@
...
+WorldsHolder.ERROR_NO_GROUPS_FILE=Groups file for world '%s' does not exist: %s
+WorldsHolder.ERROR_NO_USERS_FILE=Användarfilen för världen %s finns inte: %s
+WorldsHolder.MIRRORING_ERROR=Mirroring error with %s. Rekursiv slinga upptäcktes\!
+WorldsHolder.NEWER_GROUPS_FILE_LOADING=Nyare gruppfil hittad (laddar ändringar)\!
+WorldsHolder.NEWER_USERS_FILE_LOADING=Nyare användarfil hittad (laddar in ändringar)\!
+WorldsHolder.NO_DATA=Ingen data för %s.
+WorldsHolder.UNKNOWN_MIRRORING_FORMAT=Obekant speglingsformat för %s
+WorldsHolder.WARN_NEWER_GROUPS_FILE_UNABLE=Nyare gruppfil hittades för %s, men vi har lokala ändringar\!
+WorldsHolder.WARN_NEWER_USERS_FILE_UNABLE=Nyare användarfil hittades för %s, men vi har lokala ändringar\!
+WorldsHolder.WHAT_HAPPENED=WHAT HAPPENED?
```
Comment (l1ttleO) [inline src/languages/swedish.properties:189]: Untranslated

File: src/languages/swedish.properties [hunk 6]
```diff
@@ -0,0 +1,272 @@
...
+WorldsHolder.WORLD_LOAD_SUCCESS=Framgångsrik laddning av världen %s...
+WorldsHolder.WORLD_NOT_FOUND_DEFAULT=Den begärda världen %s har inte hittats eller är dåligt speglad. Återger standardvärlden...
+WorldsHolder.WORLD_NOT_FOUND_UNNAMED=Den begärda världen %s har inte hittats eller är dåligt speglad. Returnerar all_unnamed_worlds...
+
+# GlobalGroups Text
+GlobalGroups.BAD_FORMATTED=Den globala gruppen %s är felaktigt formaterad.
+GlobalGroups.ERROR_NEWER_GG_FOUND=En nyare GlobalGroups-fil hittades, men vi har lokala ändringar\!
+GlobalGroups.INVALID_GROUP_NAME=Invalid group name for GlobalGroup entry \#[%d] in file: %s
+GlobalGroups.INVALID_PERMISSION_NODE=Invalid permission node in global group: %s
+GlobalGroups.UNKNOWN_PERMISSION_TYPE=Unknown type of permission node in global group: %s
```
Comment (l1ttleO) [inline src/languages/swedish.properties:200]: Untranslated

File: src/languages/swedish.properties [hunk 7]
```diff
@@ -0,0 +1,272 @@
...
+GroupManager.ERROR_LOADING=Ett fel inträffade vid laddning av GroupManager
+GroupManager.ERROR_SCHEDULING_SUPERPERMS=Kan inte schemalägga superperms Uppdatering.
+GroupManager.FILE_CORRUPT=Följande fil är skadad: %s
+GroupManager.PERMISSION_DIRECT_ACCESS=%s har redan direkt tillgång till den behörigheten.
+GroupManager.PERMISSION_DIRECT_ACCESS_ALREADY=%s har redan direkt åtkomst till behörigheten.
+GroupManager.PERMISSION_EXCEPTION=%s har redan ett undantag för den här noden.
+GroupManager.PERMISSION_EXCEPTION_ALREADY=%s har redan ett undantag för den här noden.
+GroupManager.PERMISSION_MATCHING_NEGATION=%s har redan en matchande negerad nod.
+GroupManager.PERMISSION_MATCHING_NEGATION_ALREADY=%s har redan en matchande negerad nod.
+GroupManager.PERMISSION_NODE=Node:
```
Comment (l1ttleO) [inline src/languages/swedish.properties:221]:
```suggestion
GroupManager.PERMISSION_NODE=Nod:
```

File: src/languages/swedish.properties [hunk 8]
```diff
@@ -0,0 +1,272 @@
...
+GroupManager.REFRESHED=Datafiler uppdaterade.
+GroupManager.SCHEDULED_DATA_SAVING_DISABLED=Scheduled Data Saving is Disabled\!
+GroupManager.SCHEDULED_DATA_SAVING_SET=Scheduled Data Saving är inställt för varje %d minut\!
+
+# Uppgifter
+BukkitPermsUpdateTask.BUKKIT_PERMISSIONS_UPDATED=Bukkit Permissions Updated\!
```
Comment (l1ttleO) [inline src/languages/swedish.properties:227]: Untranslated

File: src/languages/swedish.properties [hunk 9]
```diff
@@ -0,0 +1,272 @@
...
+MANUCLEARP_SYNTAX=\ (/manuclearp <user>)
+MANUDEL_SYNTAX=\ (/manudel <user>)
+MANUDELP_SYNTAX=\ (/manudelp <user> <permission> [permission2] [permission3]...)
+MANUDELSUB_SYNTAX=\ (/manudelsub <user> <group>)
+MANUDELV_SYNTAX=\ (/manudelv <user> <variabel>)
+MANULISTP_SYNTAX=\ (/manulistp <user> [+])
+MANULISTV_SYNTAX=\ (/manUlistv <user>)
+MANWHOIS_SYNTAX=\ (/manwhois <user>)
+MANUADDTEMP_SYNTAX=\ (/manuaddtemp <player>)
+MANUDELTEMP_SYNTAX=\ (/manudeltemp <player>)
```
Comment (l1ttleO) [inline src/languages/swedish.properties:272]:
```suggestion
MANCHECKW_SYNTAX=\ (/mancheckw <värld>)
MANDEMOTE_SYNTAX=\ (/mandemote <användare> <grupp>)
MANGADD_SYNTAX=\ (/mangadd <grupp>)
MANGADDI_SYNTAX=\ (/mangaddi <grupp1> <grupp2>)
MANGADDP_SYNTAX=\ (/mangaddp <grupp> <tillstånd>[|1d1h1m] [tillstånd2] [tillstånd3]...)
MANGADDV_SYNTAX=\ (/mangaddv <grupp> <variabel> <värde> <värde>)
MANGCHECKP_SYNTAX=\ (/mangcheckp <grupp> <behörighet>)
MANGCHECKV_SYNTAX=\ (/mangcheckv <grupp> <variabel>)
MANGCLEARP_SYNTAX=\ (/mangclearp <grupp>)
MANGDELP_SYNTAX=\ (/mangdelp <användare> <tillstånd> [tillstånd2] [tillstånd3]...)
MANGDELV_SYNTAX=\ (/mangdelv <grupp> <variabel>)
MANGDEL_SYNTAX=\ (/mangdelv <grupp>)
MANGDELI_SYNTAX=\ (/mangdeli <grupp1> <grupp2>)
MANGLISTP_SYNTAX=\ (/manglistp <grupp>)
MANGLISTV_SYNTAX=\ (/manglistv <grupp>)
MANPROMOTE_SYNTAX=\ (/manpromote <användare> <grupp>)
MANSELECT_SYNTAX=\ (/manselect <värld>)
MANUADD_SYNTAX=\ (/manuadd <användare> <grupp> | valfritt [värld])
MANUADDP_SYNTAX=\ (/manuaddp <användare> <tillstånd>[|1d1h1m] [tillstånd2] [tillstånd3]...)
MANUADDSUB_SYNTAX=\ (/manuaddsub <användare> <grupp>[|1d1h1m] [grupp2] [grupp3]...)
MANUADDV_SYNTAX=\ (/manuaddv <användare> <variabel> <värde>)
MANUCHECKP_SYNTAX=\ (/manucheckp <användare> <tillstånd>)
MANUCHECKV_SYNTAX=\ (/manucheckv <användare> <variabel>)
MANUCLEARP_SYNTAX=\ (/manuclearp <användare>)
MANUDEL_SYNTAX=\ (/manudel <användare>)
MANUDELP_SYNTAX=\ (/manudelp <användare> <tillstånd> [tillstånd2] [tillstånd3]...)
MANUDELSUB_SYNTAX=\ (/manudelsub <användare> <grupp>)
MANUDELV_SYNTAX=\ (/manudelv <användare> <variabel>)
MANULISTP_SYNTAX=\ (/manulistp <användare> [+])
MANULISTV_SYNTAX=\ (/manUlistv <användare>)
MANWHOIS_SYNTAX=\ (/manwhois <användare>)
MANUADDTEMP_SYNTAX=\ (/manuaddtemp <spelare>)
MANUDELTEMP_SYNTAX=\ (/manudeltemp <spelare>)
```

**Review by l1ttleO — CHANGES_REQUESTED** [review]

**Review by l1ttleO — APPROVED** [review]

**Comment by ItsLynix** [comment]
Seems to me like this is mostly complete now, you may edit whatever is needed to ensure that things like placeholders parse correctly and such. Thanks for your co-operation.

**Pull request merged**